app/models/legacy/loan_question.rb

```diff
@@ -7,6 +7,25 @@ class LoanQuestion < ActiveRecord::Base
 
     remove_method :id, :question
 
+    # Helper method to sanitize local mysql db until we have cleaned up production data.
+    def self.clean_up_legacy_db
+      # mirror new group and order fields for post analysis questions
+      connection.execute("UPDATE LoanQuestions set NewGroup = Grupo where active = 3")
+      connection.execute("UPDATE LoanQuestions set NewOrder = Orden where active = 3")
+
+      # tweak so that cross-linked children will be marked as 'is_active'
+      connection.execute("UPDATE LoanQuestions set Active = 4 where Active = 2 and NewGroup is not null")
+
+      # filter out questions which seemed to be ignored by old system.
+      connection.execute("UPDATE LoanQuestions set Active = 0 where NewOrder = 0")
+      connection.execute("UPDATE LoanQuestions set Active = 0 where NewGroup is null and Type != 'Grupo'")
+      # ignore some random cruft from lastest prod db
+      connection.execute("UPDATE LoanQuestions set Active = 0 where Type = ''")
+
+      # create some test data for the 'inactive' state
+      connection.execute("update LoanQuestions set NewGroup = Grupo where active = 2 and id > 100")
+    end
```

Comment:
The first two statements here are all I did on the production db. These other changes seem like they might mess up the old system. Can they be made in the migration?